packages/dashboard/src/components/MultiCountChart/MultiCountStyles.ts

```diff
@@ -0,0 +1,40 @@
+import { IMultiCountChartStyles, IMultiCountStyleProps } from './MutliCountChart.types';
+
+export const getStyles = (props: IMultiCountStyleProps): IMultiCountChartStyles => {
+  return {
+    root: {
+      display: 'flex',
+      lineHeight: '33px',
+      flexDirection: 'row',
+      flex: 1
```

Comment:
Adding add `align-items: baseline` in here should fix vertical alignment issues in the text.